### ISSUE TITLE
Feature Request - rename files after replace of token

### ISSUE THREAD

**Issue by ckruegerkpmg** [qetza/vsts-replacetokens-task#114]
Hi,

I've a proposal, add an option for automatically rename the file after replacing the token.

We are doing the following, in our source code repo there are 2 files checked in:
server.config
server.config.token

The server.config contains the values for the local development (contains no tokens).
The server.config.token file contains tokens for the different stages.
After the tokens are replaced the server.config.token file is "renamed" to server.config.
(it's no rename but a replace)

We are running this file after tokenization but it would be nice if that's an option directly in the extension.

```
-----RenameTokenFiles.ps1------
param (
[Parameter(
   Position=0,
   Mandatory=$true,
   ValueFromPipeline=$false
)][string]$path
)

$files=get-childitem $path\*.token -recurse
foreach ($file in $files)
{
  copy $file.Fullname ($file.Fullname -replace '\.token$','') -force -verbose
}
-----RenameTokenFiles.ps1------
```

**Comment by jonmartin136** [comment]
If your .config are XML based then why not have tokens in XDT transform files? You'd use the Replace Token task to update the XDT transform files and then use the XDT Transform task (https://github.com/qetza/vsts-xdttransform-task) to update your .config files. Loads of ways to do this as well: single generic transform files and/or env-specific transform file for when an env is needing further changes compared to others.

**Comment by qetza** [comment]
Hi @ckruegerkpmg,

As @jonmartin136 said if you are using xml files you could use the `xdt transform` task to inject tokens in your `server.config` and then replace token in this file.

But as this is not the first time i hear this request i will add it to the backlog so that it will be possible to define an output file using the syntax:
`folder\server.config.token => folder\server.config`

or with wildcard:
`**\*.config.token => *.config` (the * in the output will be replaced by the value of the * in the input)

**Comment by qetza** [comment]
Hi,
I'm pleased to announce that this feature was just released in v3.3.0 :)

**Comment by ckruegerkpmg** [comment]
Thank you very much!
